test/integration.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+const assert = require('assert');
+const blc = require('broken-link-checker');
+const test = require('selenium-webdriver/testing');
+const webdriver = require('selenium-webdriver');
+
+const timeOut = 35000;
+
+const baseURL = process.env.VS_TEST_URL || 'https://viewsourceconf-stage.us-west.moz.works';
+console.log(`Running integration tests against ${baseURL}`)
+
+describe('The site...', function() {
+    this.timeout(timeOut);
+    const url = baseURL;
+
+    it('should not have any broken links', function(done) {
+        const options = {
+            // https://github.com/stevenvachon/broken-link-checker
+            excludedKeywords: ['/2015', 'livereload'],
+            filterLevel: '3',
+        };
+
+        const errors = {};
+
+        const siteChecker = new blc.SiteChecker(options, {
+            link(result) {
+                if (result.broken) {
+                    errors[result.base.resolved] = errors[result.base.resolved] || [];
+                    errors[result.base.resolved].push(
+                        `${result.url.resolved} (${blc[result.brokenReason]})`
+                    );
+                }
+            },
+            site() {
+                if (Object.keys(errors).length > 0) {
+                    const msg = `Site has broken links: \n ${JSON.stringify(errors)}`;
+                    return done(new Error(msg));
+                }
+                return done();
+            },
+        });
+
+        siteChecker.enqueue(url, url);
+    });
+});
+
+test.describe('The landing page...', function() {
+    this.timeout(timeOut);
+    const url = baseURL;
+    const expectedTitle = 'View Source Conference 2016';
+
+    test.it(`should be called "${expectedTitle}"`, function(done) {
+        const driver = new webdriver.Builder()
+            .forBrowser('firefox')
+            .build();
+
+        driver.get(url).then(function() {
+            driver.getTitle().then(function(title) {
+                assert.equal(title, expectedTitle)
```

Comment:
My main idea with this test and many of the others below is this:

* Our unit tests verify that all present URLs are valid
* But we rebuild the entire site from source with every deploy
* I want to be sure that key pages have some valid HTML on them -- something to prove that the build worked as it should have.

@stephaniehobson suggested finding something in the footer, because if the footer is there then the build must've completed and the code must be render-able. So that's where I was headed with this kind of test: if there's a title, the build didn't catastrophically fail.

Instead of this kind of test, maybe it's better to make sure a couple critical paths work: 

1. Front page has conference page links, conference page links point to valid URLs, content at those URLs includes a register button, register button points to a valid URL.
2. Conference page has a CFP link, CFP link points to a valid URL.
3. Conference page has a newsletter link, newsletter signup works.
4. Conference page has a sponsor prospectus link; sponsor prospectus link is valid.
5. Code of Conduct has some real content we expect (this won't be changing like the rest of the site will).

We're blanket testing URLs, so in all of the above link tests we mostly just need to verify visibility of links.

If this plan sounds good I can make these changes fairly easily.